sql/promscale--0.0.0.sql

```diff
@@ -17,7 +17,19 @@ BEGIN
 END;
 $stop_bgw$;
 
-DROP TABLE public.prom_schema_migrations;
+DO
+$drop_prom_schema_migrations$
+DECLARE
+    schema TEXT;
+BEGIN
+    -- The prom_schema_migrations table was not schema-qualified on creation,
+    -- so it could be in any schema. Find out the schema it's in and drop.
+    SELECT schemaname INTO STRICT schema FROM pg_catalog.pg_tables WHERE tablename = 'prom_schema_migrations';
+    IF schema IS NOT NULL THEN
```

Comment:
FYI, this line is a no-op.

Reply:
Why?

Reply:
`STRICT` in the line above will error out if nothing is returned.

Reply:
Aha, indeed. Thank you!

Reply:
Are we ok with the execution stopping here if we don't find the table?